.github/workflows/release.yaml

```
@@ -51,7 +51,7 @@ jobs:
 
     - name: Upload artifacts
       # https://github.com/actions/upload-artifact/commit/65462800fd760344b1a7b4382951275a0abb4808
-      uses: actions/upload-artifact@834a144ee995460fba8ed112a2fc961b36a5ec5a
+      uses: actions/upload-artifact@b18b1d32f3f31abcdc29dee3f2484801fe7822f4 # v3.2.1
```

This commit points to the current `main` branch which isn't the same as v3.2.1.

The previous commit pointed to [v4.3.6](https://github.com/actions/upload-artifact/releases/tag/v4.3.6) so I suggest we bump to [v4.4.0](https://github.com/actions/upload-artifact/releases/tag/v4.4.0) which is the latest release in the v4.4 line.

```suggestion
      uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
```